sympy/solvers/tests/test_solvers.py

```diff
@@ -1351,25 +1350,6 @@ def test(e1, e2):
     assert solve(((.7 + c)/tot - .6, (.2 + z)/tot - .3, t/tot - .1)) == []
 
 
-def test_check_assumptions():
-    x = symbols('x', positive=True)
-    assert solve(x**2 - 1) == [1]
```

Comment:
This test for solve seems to be not replaced anywhere

Reply:
I will see if there is another test where results are limited by symbol assumptions.